recipes/hdmf/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "hdmf" %}
+{% set version = "0.post0.dev1081" %}
+{% set sha256 = "8b0264a51ef1aea5fb22d450a7fa6cbc74bac97421e9667d1b7caee0ab74ce57" %}
+{% set PYTHON = "python" %}
```

Comment:
conda-build is supposed to set this to the full python path. You shouldn't need to do this.

Reply:
Before I added that line, the Circle build failed for this reason: 

```
Error: Failed to render jinja template in /home/conda/conda-recipes/hdmf/meta.yaml:
'PYTHON' is undefined
Exited with code 1
```

https://circleci.com/gh/conda-forge/staged-recipes/34924?utm_campaign=workflow-failed&utm_medium=email&utm_source=notification